spec/features/managing_peeps_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+require_relative 'helpers/session'
+include SessionHelpers
+
+feature 'User can' do
+
+
+  before(:each) do
+    @user = User.create(:name => 'Anna Klimas',
+              :username => 'ania2ab',
+              :email => 'anna@klimas.com',
+              :password => 'apple',
+              :password_confirmation => 'apple')
+    @peep = Peep.create(message: 'Hello world',
+      user_id: 'ania2ab', created_at: 'Time.new.strftime("%F %H:%M")')
+
+  end
+
+  scenario 'add peeps' do
+    expect(Peep.count).to eq(0)
+    sign_in("anna@klimas.com", "apple")
+    visit '/'
+    click_link 'Click to add new peep'
+    add_peep('This is my first peep')
```

Comment:
`add_peep` starts with a `visit`, so the `visit` and `click_link` steps are irrelevant.